frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -8,29 +10,54 @@ const formatCount = (count: number) => {
   return `${hCount.string} Object${count === 1 ? '' : 's'}`;
 };
 
-const BucketsRowStatus: React.FC<BucketsRowStatusProps> = React.memo(({ status, link, error }) => (
-  <div className="nb-buckets-card__row-status-item">
-    {error || _.isNil(status) ? (
-      <span className="co-dashboard-text--small nb-buckets-card__row-subtitle">Not available</span>
-    ) : Number(status) > 0 ? (
-      <>
-        <a href={link} style={{ textDecoration: 'none' }} target="_blank" rel="noopener noreferrer">
-          <RedExclamationCircleIcon className="co-dashboard-icon nb-bucket-card__status-icon" />
+const BucketsStatus: React.FC<{ link: string; status: number }> = React.memo(({ link, status }) => {
+  return (
+    <div className="nb-buckets-card__status">
+      <a href={link} style={{ textDecoration: 'none' }} target="_blank" rel="noopener noreferrer">
```

Comment:
use what ?

Reply:
Link should not be used for external links

Reply:
use ExternalLink

Reply:
Not possible. Please check the definition of ExternalLink. Its only for strings

Reply:
can you update the type for text for it to accept Components. @rawagner wdyt?

Reply:
With latter approach it will be a significant change since ExternalLink is used at various places. Also I feel we can avoid that here since adding ExternalLink would not do much but just replace `a` with ExternalLink plus its just one case here. Plus, did not find this anywhere else in console

Reply:
With latter approach it will be a significant change since ExternalLink is used at various places. Also I feel we can avoid that here since adding ExternalLink would not do much but just replace `a` with `ExternalLink` plus its just one case here. Plus, did not find this anywhere else in console